arch/Kconfig

```diff
@@ -269,6 +269,14 @@ config FLOAT
 	  Disabling this option means that any thread that uses a
 	  floating point register will get a fatal exception.
 
+config FP_DOUBLE_PRECISION
+	bool
+	# Omit prompt to signify "hidden" option
+	depends on FLOAT
+	help
+	  This option signifies the use of an FPU supporting double
```

Comment:
Does it signify the "use"? Or, perhaps, the support? 
That said: could we reword the test to align with the description of CPU_HAS_FPU? But, no strong opinion on this .

Reply:
Ok, I'll try to reword the description.